time.c

```diff
@@ -3777,25 +3777,28 @@ time_zonelocal(VALUE time, VALUE off)
 
 /*
  *  call-seq:
- *     time.localtime -> time
- *     time.localtime(utc_offset) -> time
+ *    localtime -> self or new_time
+ *    localtime(zone) -> new_time
  *
- *  Converts _time_ to local time (using the local time zone in
- *  effect at the creation time of _time_) modifying the receiver.
+ *  With no argument given:
  *
- *  If +utc_offset+ is given, it is used instead of the local time.
+ *  - Returns +self+ if +self+ is a local time.
+ *  - Otherwise returns <tt>localtime(self.local)</tt>:
```

Comment:
There's no method called `#local` for `Time`. I don't think there's an easy way to get the user's timezone (a quick search suggests `Time.now.zone`, but IMO that's rather awkward to use). Maybe we can just say "Otherwise returns a new Time in the user's local timezone".

Reply:
Fixed.